### ISSUE TITLE
Bug in amp task poselib rotation3d `rot_matrix_from_quaternion`

### ISSUE THREAD

**Issue by thomas-rkk** [NVIDIA-Omniverse/IsaacGymEnvs#233]
The rotation matrix returned is not normalized or orthogonal.
Here: https://github.com/isaac-sim/IsaacGymEnvs/blob/aeed298638a1f7b5421b38f5f3cc2d1079b6d9c3/isaacgymenvs/tasks/amp/poselib/poselib/core/rotation3d.py#L419
Should have been:

```
R2 = torch.stack([R20, R21, R22], dim=-1)
```
